yash-semantics/src/command/pipeline.rs

```diff
@@ -145,11 +145,12 @@ async fn execute_job_controlled_pipeline(
         }
         Err(errno) => {
             // TODO print error location using yash_env::io::print_error
-            env.print_error(&format!(
-                "cannot start a subshell in the pipeline: {}\n",
-                errno.desc()
-            ))
-            .await;
+            env.system
+                .print_error(&format!(
+                    "cannot start a subshell in the pipeline: {}\n",
+                    errno.desc()
+                ))
+                .await;
             Break(Divert::Interrupt(Some(ExitStatus::NOEXEC)))
```

Comment:
Thanks, but the TODO comment still needs to be addressed.

Reply:
Oh, sorry for the confusion. I meant that the issue mentioned in the comment was yet to be handled, so we need to retain the comment for now.

The current code uses the `SharedSystem::print_error` method for printing the message, but we should replace it with the `yash_env::io::print_error` function in the future. The reason we're not doing that now is because the location information we want to pass to `yash_env::io::print_error` is not available in the current context. We need another refactoring to make the location usable, but that's not the scope of this pull request.